src/09_dictionaries.py

```diff
@@ -35,10 +35,18 @@
 
 # Add a new waypoint to the list
 # YOUR CODE HERE
+new = {
+    "lat": 36,
+    "lon": -119,
+    "name": "my place"
+}
+waypoints.append(new.copy())
```

Comment:
That's cool that you are thinking about passing a new object rather than referencing.